File: Kotlin/KT-74151/trigger-program-after-minimization.kt
// Kotlin Classes
interface I0 {
    fun func(arg0: ArrayList<Any>)
}

interface I1: I0 {
    override fun func(arg0: ArrayList<Any>) {
        println("I1")
    }
}
abstract class A1: I0 {
    override fun func(arg0: ArrayList<Any>) {
        println("A1")
    }
}

class A3: A2<Any>(), I1, I0 // could not compile. MANY_IMPL_MEMBER_NOT_IMPLEMENTED
// Java Class
public class A2<T> extends A1 implements I1 {}

File: Scala2/scala2-13074/trigger-program-after-minimization.scala
// FILE: I0.java
public interface I0<T> {
  public void func(A<String> a, T s);
}
// FILE: I1.java
public interface I1 extends I0<String> {
  public default void func(A<Object> a, String s) {

  }
}
// Scala Classes
class A[T0]
class A0 extends I1 {
}